packages/react-components/components/PhoneNumberInput.test.tsx

```diff
@@ -1,100 +1,41 @@
 import PhoneNumberInput from '@celo/react-components/components/PhoneNumberInput'
-import { shallow } from 'enzyme'
 import * as React from 'react'
-import { Text } from 'react-native'
-import Autocomplete from 'react-native-autocomplete-input'
-import * as renderer from 'react-test-renderer'
+import { fireEvent, render } from 'react-native-testing-library'
 
 describe('PhoneNumberInput', () => {
-  it('renders correctly with minimum props', () => {
-    const tree = renderer.create(
-      <PhoneNumberInput
-        defaultCountry={'USA'}
-        setE164Number={jest.fn()}
-        setIsValidNumber={jest.fn()}
-        setCountryCode={jest.fn()}
-      />
-    )
-    expect(tree).toMatchSnapshot()
-  })
-
   describe('when defaultCountry is falsy', () => {
-    it('renders an AutoComplete', () => {
-      const numberInput = shallow(
+    it('renders an AutoComplete and a country can be selected', () => {
+      const mockSetCountryCode = jest.fn()
+      const { getByTestId, toJSON } = render(
         <PhoneNumberInput
           defaultCountry={null}
           setE164Number={jest.fn()}
           setIsValidNumber={jest.fn()}
-          setCountryCode={jest.fn()}
-          inputCountryPlaceholder="Nations"
+          setCountryCode={mockSetCountryCode}
         />
       )
-      expect(numberInput.find(Autocomplete).props()).toEqual(
```

Comment:
Note: from all this red it may look like i removed tests but actually this simplified versions tests just as much :)